app/controllers/alchemy/admin/resources_controller.rb

```diff
@@ -58,7 +58,7 @@ def update
         resource_instance_variable.update_attributes(resource_params)
         render_errors_or_redirect(
           resource_instance_variable,
-          resources_path,
+          resources_path(resource_handler.resources_name, q: params[:q]),
```

Comment:
`resources_path` takes two default parameters, and if I only pass in a hash, it'll think the hash is the resources_path. I could also go and refactor it to uses named arguments, but I think that's outside the scope of this PR.